runtime/v2/bundle.go

```diff
@@ -121,7 +121,7 @@ func (b *Bundle) Delete() error {
 	if err := mount.UnmountAll(rootfs, 0); err != nil {
 		return errors.Wrapf(err, "unmount rootfs %s", rootfs)
 	}
-	if err := os.Remove(rootfs); err != nil && os.IsNotExist(err) {
+	if err := os.Remove(rootfs); err != nil && !os.IsNotExist(err) {
```

Comment:
Curious; do we know if there's a reason we didn't use `atomicDelete()` for this as well? https://github.com/containerd/containerd/blob/433662502f428773746fe403484d61452e77f7e2/runtime/v2/bundle.go#L146

Reply:
I think the `rootfs` is empty dir which is used to be mount point. No need to use atomicDelete